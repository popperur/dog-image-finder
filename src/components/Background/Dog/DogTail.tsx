import {
  DogTailContainer,
  DogTailLeft,
  DogTailMiddle,
  DogTailRight,
  DogTailTop,
} from "components/Background/Dog/styles.tsx";

function DogTail() {
  return (
    <DogTailContainer>
      <DogTailLeft />
      <DogTailMiddle />
      <DogTailRight />
      <DogTailTop />
    </DogTailContainer>
  );
}

export default DogTail;
