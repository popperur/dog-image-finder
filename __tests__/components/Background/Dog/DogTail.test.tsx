import { render } from "@testing-library/react";
import { describe, it, expect } from "vitest";
import DogTail from "components/Background/Dog/DogTail";

describe("DogTail component", () => {
  it("matches the snapshot", () => {
    const { asFragment } = render(<DogTail />);

    expect(asFragment()).toMatchSnapshot();
  });
});
