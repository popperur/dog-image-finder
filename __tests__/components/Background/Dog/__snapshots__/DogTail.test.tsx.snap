// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`DogTail component > matches the snapshot 1`] = `
<DocumentFragment>
  .c0 {
  position: absolute;
  left: 0;
  bottom: 0;
  width: 24.5px;
  height: 114px;
  background-color: #ff9a29;
}

.c1 {
  position: absolute;
}

.c4 {
  top: -38px;
  left: -28px;
  width: 12px;
  height: 70px;
  border-radius: 0 32px 0 0;
  border: 22px solid transparent;
  border-right-color: #ff9a29;
  transform: skew(0deg) rotate(-1deg) rotatey(1deg) skewy(-26deg);
}

.c3 {
  top: -23px;
  left: -27px;
  width: 14px;
  height: 9px;
  border-radius: 1000px;
  border: 12px solid transparent;
  border-right-color: #ff9a29;
  transform: skew(-7deg) rotate(-74deg) rotatey(1deg) skewy(-2deg) scale(2);
  animation: dPwbxO 150ms linear infinite;
}

.c2 {
  top: -34px;
  left: -15px;
  width: 0;
  height: 14px;
  border-radius: 0 32px 0 0;
  border: 15px solid transparent;
  border-right-color: #ff9a29;
}

.c5 {
  top: -43px;
  left: -27px;
  width: 20px;
  height: 24px;
  background-color: #f1f1f2;
  border-top-left-radius: 20px;
  border-bottom-left-radius: 20px;
  transform: skew(-2deg) rotate(-12deg);
  animation: hvbQKD 150ms linear infinite;
}

<div
    class="c0"
  >
    <div
      class="c1 c2"
    />
    <div
      class="c1 c3"
    />
    <div
      class="c1 c4"
    />
    <div
      class="c1 c5"
    />
  </div>
</DocumentFragment>
`;
